phase1/vsphere/configure-vm.sh

```diff
@@ -37,15 +61,55 @@ echo "${nodes_dns_mappings}" >> /etc/hosts
 
 
 systemctl enable docker
+if [ $? -ne 0 ]; then
+    echo "Failed to enable docker"
+    exit 1
+fi
 systemctl start docker
+if [ $? -ne 0 ]; then
+    echo "Failed to start docker"
+    exit 1
+fi
 docker run \
   --net=host \
   -v /:/mnt/root \
   -v /run:/run \
   -v /etc/kubernetes:/etc/kubernetes \
   -v /var/lib/ignition:/usr/share/oem \
   "${installer_container}" /bin/do_role
+if [ $? -ne 0 ]; then
+    echo "Failed to docker run installer container"
+    exit 1
+fi
 systemctl daemon-reload
+if [ $? -ne 0 ]; then
+    echo "Failed to reload daemon"
+    exit 1
+fi
+docker pull \
```

Comment:
I would suggest to do this before running ignition container since there is no point in running ignition container if the hyperkube image doesn't exist.